### DISCUSSION TITLE
Starting my Terminal client has Oh My Posh calling home to api.github.com

### DISCUSSION THREAD

**Discussion by BabyFnord** [JanDeDobbeleer/oh-my-posh#3707] [General]
Dear Devs,

I love _Oh My Posh_ but, I dislike that it does unsolicited calls to api.github.com, any time I fire up my Terminal client. I haven't found an elegant way to disable this unwanted feature, other than have my network filter block _Oh My Posh_ from contacting api.github.com. 
![Bildschirmfoto 2023-04-12 um 10 10 18](https://user-images.githubusercontent.com/39904614/231401455-7d680a0e-0b6f-4f68-a00b-0ce19f33ac9d.png)
Is this some kind of metrics/telemetry? Is there a way to permanently disable this behaviour? 
I really don't need version checks, my weekly update routine caters for this. 

All best,
BF

**Comment by JanDeDobbeleer** [comment]
Yes, you can run `oh-my-posh disable notice`. All it does is fetch the latest version to see if you can upgrade.

**Comment by BabyFnord** [comment]
Thank you, @JanDeDobbeleer ! Is `oh-my-posh disable notice` persistent across updates?

**Reply by JanDeDobbeleer** [reply]
@BabyFnord should be, it's stored in the local cache which is outside of the installation artifacts.

**Comment by BabyFnord** [comment]
Thanks a bunch, @JanDeDobbeleer 🙂.

**Comment by cvanelteren** [comment]
#6998 I tried this but this does not seem to work for me.
